ui-lib.css

```diff
@@ -511,10 +511,18 @@ header .logo {
   margin-right: 5px;
   vertical-align: middle;
 }
+
+.acc-pane > h3:focus {
+  outline: none;
```

Comment:
This removes the default browser outline to indicate the focus on the page. I haven't seen this default outline elsewhere in Wix.